content/pt-br/docs/tasks/access-application-cluster/access-cluster-services.md

```diff
@@ -0,0 +1,124 @@
+    - Proxies podem causar problemas para algumas aplicações web.
+    - Só funciona para HTTP/HTTPS.
+    - Descrito [aqui](#manually-constructing-apiserver-proxy-urls).
+  - Acesse a partir de um nó ou Pod no cluster.
+    - Execute um Pod e, em seguida, conecte-se a um shell nele usando [kubectl exec](/docs/reference/generated/kubectl/kubectl-commands/#exec). Conecte-se a outros nós, Pods e serviços a partir desse shell.
+    - Alguns clusters podem permitir que você faça ssh para um nó no cluster. De lá, você pode ser capaz de acessar os serviços do cluster. Este é um método não padrão e funcionará em alguns clusters, mas não em outros. Navegadores e outras ferramentas podem ou não estar instalados. O DNS do cluster pode não funcionar.
+
+### Descobrindo serviços integrados
+
+Normalmente, existem vários serviços que são iniciados em um cluster pelo kube-system. Obtenha uma lista desses serviços com o comando `kubectl cluster-info`:
+
+```shell
+kubectl cluster-info
+```
+
+A saída é semelhante a esta:
+
+```
+Kubernetes master is running at https://192.0.2.1
+elasticsearch-logging is running at https://192.0.2.1/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy
+kibana-logging is running at https://192.0.2.1/api/v1/namespaces/kube-system/services/kibana-logging/proxy
+kube-dns is running at https://192.0.2.1/api/v1/namespaces/kube-system/services/kube-dns/proxy
+grafana is running at https://192.0.2.1/api/v1/namespaces/kube-system/services/monitoring-grafana/proxy
+heapster is running at https://192.0.2.1/api/v1/namespaces/kube-system/services/monitoring-heapster/proxy
+```
+
+Isso mostra a URL proxy-verbo para acessar cada serviço. Por exemplo, este cluster tem o registro em nível de cluster habilitado (usando o Elasticsearch), que pode ser acessado em `https://192.0.2.1/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/` se as credenciais adequadas forem passadas ou através de um proxy kubectl, por exemplo: `http://localhost:8080/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/`.
+
+{{< note >}}
+Consulte [Acessando clusters usando a API Kubernetes](/docs/tasks/administer-cluster/access-cluster-api/#accessing-the-cluster-api) para obter informações sobre como passar credenciais ou usar o proxy kubectl.
+{{< /note >}}
+
+### Construindo manualmente URLs de proxy do apiserver
+
+Como mencionado acima, você usa o comando `kubectl cluster-info` para recuperar a URL do proxy do serviço. Para criar URLs de proxy que incluem pontos de extremidade, sufixos e parâmetros de serviço, você adiciona à URL do proxy do serviço:
+"`http://`*`endereço_do_mestre_do_kubernetes`*`/api/v1/namespaces/`*`nome_do_namespace`*`/services/`[https:]nome_do_serviço[:nome_da_porta]`*`/proxy`"
```

Comment:
```suggestion
Como mencionado acima, você usa o comando `kubectl cluster-info` para recuperar a URL do proxy do serviço. Para criar URLs de proxy que incluem _endpoints_, sufixos e parâmetros de serviço, você adiciona à URL do proxy do serviço:
`http://`*`endereço_do_mestre_do_kubernetes`*`/api/v1/namespaces/`*`nome_do_namespace`*`/services/`*`[https:]nome_do_serviço[:nome_da_porta]`*`/proxy`
```

I'm unfamiliar with a good translation of endpoint to Portuguese. I lean towards keeping it as _endpoint_, since it's well-known terminology for APIs.

There were also no enclosing quotes in the original text, and there's a missing few format signs that's currently breaking rendering.